integration_tests/pages/refer/oasysConfirmed.ts

```diff
@@ -14,6 +14,13 @@ export default class OasysConfirmedPage extends Page {
     this.referral = referral
   }
 
+  confirmOasys() {
+    cy.get('[name="oasysConfirmed"]').check()
+    this.referral = { ...this.referral, oasysConfirmed: true }
+    cy.task('stubReferral', this.referral)
+    this.shouldContainButton('Save and continue').click()
+  }
```

Comment:
Add comment in the code and commit message to say that we're stubbing the referral here to make sure the updated referral is available on the task list page